python-main.js

```diff
@@ -111,6 +133,67 @@ function pythonEditor(id) {
+            "MicroBitAnalogDigitalPin" : ["read_analog", "read_digital", "set_analog_period", "set_analog_period_microseconds", "write_analog", "write_digital"],
+            "MicroBitAnalogPin" : ["read_analog", "read_digital", "set_analog_period", "set_analog_period_microseconds", "write_analog", "write_digital"], 
+            "MicroBitDigitalPin" : ["read_digital", "write_digital"],
+            "MicroBitTouchPin" : ["is_touched", "read_analog", "read_digital", "set_analog_period", "set_analog_period_microseconds", "write_analog", "write_digital"],
+            "accelerometer" : ["current_gesture", "get_gestures", "get_values", "get_x", "get_y", "get_z", "was_gesture"],
+            "button_a" : ["get_presses", "is_pressed", "was_pressed"],
+            "button_b" : ["get_presses", "is_pressed", "was_pressed"],
+            "compass" : ["calibrate", "clear_calibration", "get_field_strength", "get_x", "get_y", "get_z", "heading", "is_calibrated"],
+            "display" : ["clear", "get_pixel", "is_on", "off", "on", "read_light_level", "scroll", "set_pixel", "show"],
+            "i2c" : ["init", "read", "scan", "write"],
+            "panic" : "",
+            "reset" : "",
+            "running_time" : "",
+            "sleep" : "",
+            "spi" : ["init", "read", "write", "write_readinto"],
+            "temperature" : "",
+            "uart" : ["any", "init", "read", "readall", "readline", "write"]
+        },
+        "audio" : ["play", "AudioFrame"],
+        "machine" : ["disable_irq", "enable_irq", "freq", "reset", "time_pulse_us", "unique_id"],
+
+        "micropython" : ["const", "heap_lock", "heap_unlock", "kbd_intr", "mem_info", "opt_level", "qstr_info", "stack_use"],
+
+        "music" : ["BADDY", "BA_DING", "BIRTHDAY", "BLUES", "CHASE", "DADADADUM", "ENTERTAINER", "FUNERAL", "FUNK", "JUMP_DOWN", "JUMP_UP", "NYAN", "ODE", "POWER_DOWN", "POWER_UP", "PRELUDE", "PUNCHLINE", "PYTHON", "RINGTONE", "WAWAWAWAA", "WEDDING", "get_tempo", "pitch", "play", "reset", "set_temp", "stop"],
+
+        "speech" : ["pronounce", "say", "sing", "translate"],
+
+        "radio" : ["RATE_1MBIT", "RATE_250KBIT", "RATE_2MBIT", "config", "off", "on", "receive", "receive_bytes", "receive_bytes_into", "receive_full", "reset", "send", "send_bytes"]
+    };
```

Comment:
There are other modules missing:

```
>>> help('modules')
__main__          love              os                time
antigravity       machine           radio             ucollections
array             math              random            ustruct
audio             microbit          speech            utime
builtins          micropython       struct
collections       music             sys
gc                neopixel          this
Plus any modules on the filesystem
>>> 
```

Minus `antigravity`, `this`, `love`, which are kind of easter eggs.